### ISSUE TITLE
problems installing nvda in windows 11 insider preview for ARM

### ISSUE THREAD

**Issue by michalzablocki319** [nvaccess/nvda#13329]
I can't install nvda in windows 11 for arm. When I launch the installer it gives me an NSI error loading installer and there's nothing I can do about it.

**Comment by michalzablocki319** [comment]
The installer doesn't launch. It gives me the NSIS error launching installer. The system is windows 11 for arm insider preview. It's a freshly installed copy without anything installed. The system is running on a m1 powered Mac. It's the newest installer 2021.3. The only software running is Microsoft narrator. I tried restarting the computer, but it doesn't fix the issue.

**Comment by seanbudd** [comment]
@michalzablocki319 have you tried an older installer?
What directory are you running the installer from?

**Comment by Brian1Gaff** [comment]
Is the NVDA current version actually supposed to work on Arm? I thought this 
was only an experimental version since Windows is quite different on Arm to 
what it is in Intel  systems.

Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Brian1Gaff** [comment]
I would have thought the Apple m1 chip would need to be running a software 
emulator for the Intel chip and then a standard windows on top of that in 
order to stand anychance at all of it working.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by lukaszgo1** [comment]
@Brian1Gaff  Yes ,NVDA on ARM is a supported scenario. In general questions like this are better suited for the users list and when posted in an unrelated issue they just detract from the main subject - please refrain from doing this in the future.

**Comment by seanbudd** [comment]
Consider solving this by upgrading NSIS: #13270

**Comment by seanbudd** [comment]
Hi @michalzablocki319  - 

Could you try installing with [this installer](https://ci.appveyor.com/api/buildjobs/yfcbf3j6vrm2by5l/artifacts/output%2Fnvda_snapshot_try-NSISAsSubmodule-24877%2Cffd63ec6.exe)?  We are trying to test if #13398 solves your problem of the installer failing.

Regards,
Sean

**Comment by seanbudd** [comment]
Can you test the latest alpha build and confirm that this is still an issue: https://www.nvaccess.org/files/nvda/snapshots/ ?

**Comment by seanbudd** [comment]
closing this issue as we have not heard back from the author
